docs/services/elections_bulletin_board.md

```diff
@@ -29,10 +26,15 @@ The following YAML snippet with all the defined environment variables should be
 
 ```yaml
   bulletin_board:
-    identification_private_key: |
-<%= ENV["BULLETIN_BOARD_IDENTIFICATION_PRIVATE_KEY"]&.indent(6) %>
+    identification_private_key: <%= ENV["BULLETIN_BOARD_IDENTIFICATION_PRIVATE_KEY"] %>
     server: <%= ENV["BULLETIN_BOARD_SERVER"] %>
     api_key: <%= ENV["BULLETIN_BOARD_API_KEY"] %>
+    number_of_trustees: <%= ENV["BULLETIN_BOARD_NUMBER_OF_TRUSTEES"] %>
+    authority_name: <%= ENV["BULLETIN_BOARD_AUTHORITY_NAME"] %>
+    scheme:
+      name: <%= ENV["BULLETIN_BOARD_SCHEME_NAME"] %>
+      parameters:
+        quorum: <%= ENV["BULLETIN_BOARD_SCHEME_QUORUM"] %>
```

Comment:
As the parameters could vary between different voting schemes, we could use the same approach for this as we do with the identification private key. I mean, storing a JSON with all the parameters and allow the administrator to add or remove keys on the env variable. E.g. it could be like this:
```yaml
    scheme:
      name: <%= ENV["BULLETIN_BOARD_SCHEME_NAME"] %>
      parameters: <%= ENV["BULLETIN_BOARD_SCHEME_PARAMETERS"] %>
```
What do you think about this option?

Reply:
@leio10 Since we will move all this to the bulletin board gem, I'd address it there. What do you think?